test/js_test.go

```diff
@@ -1582,6 +1584,7 @@ func TestJetStreamImportDirectOnly(t *testing.T) {
 			U: {
 				users: [ {user: rip, password: bar} ]
 				imports [
+					{ service: { subject: "$JS.API.INFO", account: JS } }
```

Comment:
This is needed so that it is possible to call `nc.JetStream()` and create a context to be able to publish.